nixos-modules/host/options.nix

```diff
@@ -25,6 +25,15 @@
     vms = mkOption {
       type = with types; attrsOf (submodule ({ config, name, ... }: {
         options = {
+          evaluatedConfig = mkOption {
+            description = ''
+              The evaluated configuration of this MicroVM, as a NixOS
+              module, for building **with** a flake.
```

Comment:
Is this for using `self.nixosConfigurations.*` that is usually already evaluated by `nixpkgs.lib.nixosSystem`? Could you perhaps clarify in the `description`?

Reply:
That is not the reason I need this option.

I'm using my own evaluation function for my config (https://github.com/flafydev/Combined-Manager). With this option I can use my function instead of having to use the default NixOS config eval function.

What you're saying can also be a good use case for this option.

What do you think of this description?
```
An already evaluated configuration of this MicroVM.
Allows supplying an already evaluated configuration or use an alternative configuration evaluation function instead of NixOS's default eval-config.
```

Reply:
bump

Reply:
Yeah, that's more clear.